osgp/protocol-adapter-mqtt/osgp-protocol-adapter-mqtt/src/main/resources/logback.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+  <!--
+    <appender name="DB" class="ch.qos.logback.classic.db.DBAppender">
+      <connectionSource class="ch.qos.logback.core.db.DriverManagerConnectionSource">
+        <driverClass>org.postgresql.Driver</driverClass>
+        <url>jdbc:postgresql://localhost/osgp_logging</url>
+        <user>osp_admin</user>
+        <password>1234</password>
+      </connectionSource>
+    </appender>
+  -->
+
+  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{yyyy-MM-dd HH:mm:ss.SSS}] [${HOSTNAME}] [%thread] %level %logger{36}@%method:%line - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>/var/log/osp/logs/osgp-adapter-protocol-mqtt.log</file>
+    <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+      <fileNamePattern>/var/log/osp/logs/osgp-adapter-protocol-mqtt.log.%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+      <maxFileSize>100MB</maxFileSize>
+      <maxHistory>10</maxHistory>
+      <totalSizeCap>20GB</totalSizeCap>
+    </rollingPolicy>
+    <encoder>
+      <pattern>%d{yyyy-MM-dd HH:mm:ss.SSS}] [${HOSTNAME}] [%thread] %level %logger{36}@%method:%line - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <logger name="org.springframework.ws.server.MessageTracing" level="TRACE" additivity="false">
+    <appender-ref ref="CONSOLE"/>
+  </logger>
+
+  <logger name="org.springframework.ws.client.MessageTracing" level="TRACE" additivity="false">
+    <appender-ref ref="CONSOLE"/>
+  </logger>
```

Comment:
These 2 loggers for `org.springframework.ws.server.MessageTracing` and `org.springframework.ws.client.MessageTracing` have been used during development, I assume. Either remove then or adjust the logging level to `WARN`.

Reply:
Taken literally from osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/resources/logback.xml

Will remove these loggers here, but not from the DLMS protocol adapter.